dpgen/generator/run.py

```diff
@@ -1327,6 +1385,68 @@ def run_model_devi (iter_index,
             errlog = 'model_devi.log')
         submission.run_submission()
 
+def run_model_devi(iter_index,jdata,mdata):
```

Comment:
These codes will fundamentally change `DP-GEN`'s protocol by adding a `while` loop in `run_model_devi`.  Is it possible to put all `calypso` commands to one single script, instead of `DP-GEN`'s main process?